cfgov/preprocessed/css/organisms/sidebar-breakout.less

```diff
@@ -0,0 +1,79 @@
+/* topdoc
+  name: Sidebar Breakout
+  family: cfgov-organism
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="o-sidebar-breakout">
+            <h3 class="o-sidebar-breakout_heading">
+            </h3>
+            <div class="o-sidebar-breakout_column">
+                <div class="u-flexible-container">
+                    <div class="u-flexible-container_inner"
+                         style="background-image:url();">
+                    </div>
+                </div>
+            </div>
+            <div class="o-sidebar-breakout_column">
+                <h4 class="o-sidebar-breakout_intro-heading">
+                </h4>
+                <p>{{ settings.intro.body }}</p>
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-sidebar-breakout
+            .o-sidebar-breakout_heading
+          .o-sidebar-breakout_column
```

Comment:
Indent here and below. `o-sidebar-breakout_column` is a child of `o-sidebar-breakout`, not a sibling.